### PR TITLE
✨Source Rss: Migrate Python CDK to Low-code CDK

### LINKED ISSUE #36417: ✨Source RSS: Migrate Python CDK to Low-code CDK
### Topic

Migrate RSS Source to Low-code CDK

### Relevant information

https://docs.airbyte.com/integrations/sources/rss

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
*Migrating Source Rss to Low-Code CDK*
Closes airbytehq/airbyte-internal-issues#6952

## How
*Developed using (Configuration Based Source) low-code CDK*

## Recommended reading order
1. `spec.yaml`
2. `manifest.yaml`
3. `schemas/*`

## Tests

<details open><summary><strong>Integration & Acceptance</strong></summary>

 - Full passed state:
 
image

</details>

## 🚨 User Impact 🚨
- Spec Breaking changes, migration to low-code

### CODE REVIEW

**Comment by btkcodedev** [comment]
image


The full refresh is passing
Incremental sync DateTimeCursor would not be required as it is custom parsed as incremental in custom component

Tested config URL: {
  "url": "https://www.nasa.gov/rss/dyn/breaking_news.rss"
}

Thanks!

**Review by girarda — APPROVED** [review]
small nit: I removed the reference to the expected records file because it didn't exist. Looks great otherwise!

**Comment by girarda** [comment]
@btkcodedev , there are 3 failures in CI

```
source-rss - ❌ Failed - Breaking changes must be accompanied by a migration guide: Migration guide file is missing for source-rss. Please create a migration guide in ./docs/integrations/<connector-type>s/<connector-name>-migrations.md`.
source-rss - ❌ Failed - Connectors must use Poetry for dependency management: poetry.lock file is missing.
source-rss - ❌ Failed - Connector version in metadata.yaml and pyproject.toml file must match: Version is 1.0.0 in metadata.yaml, but version is 0.1.0 in pyproject.toml. These two files have to be consistent.
Error: 3 checks failed
```

Let me know if you have questions. Otherwise, please fix them and I'll release the connector 🚀

**Comment by btkcodedev** [comment]
@girarda Resolved the issues, Will follow the same for other PRs, Please continue the merge :bowing_man:

**Comment by btkcodedev** [comment]
@girarda Resolved the issues, Please continue the merge :bowing_man: 
Latest test results: All pass
image

File: airbyte-integrations/connectors/source-rss/metadata.yaml [hunk 1]
```diff
@@ -1,29 +1,43 @@
...
+      - "*"
+  registries:
+    oss:
+      enabled: true
+    cloud:
+      enabled: true
+  releases:
+    breakingChanges:
+      1.0.0:
+        upgradeDeadline: "2024-04-15"
```
Comment (girarda) [inline airbyte-integrations/connectors/source-rss/metadata.yaml:13]: let's set this to the end of the month

**Review by girarda — COMMENTED** [review]
can you run `airbyte-ci format fix all` and commit the changes? then we'll be good to 🚢

**Comment by btkcodedev** [comment]
@girarda All comments resolved, formatted and updated
Please continue review process, 
Format results - full pass
image


arbyte-ci test results - Full pass
image

**Pull request merged**